htdocs/install/mysql/migration/22.0.0-23.0.0.sql

```diff
@@ -339,4 +339,5 @@ INSERT INTO llx_c_currencies ( code_iso, unicode, active, label ) VALUES ( 'CDF'
 
 ALTER TABLE llx_societe MODIFY COLUMN mode_reglement integer;
 
+ALTER TABLE llx_c_email_templates CHANGE COLUMN datec datec timestamp DEFAULT CURRENT_TIMESTAMP;
```

Comment:
We must abandon the use of current timestamp on fields in favor of using dol_now in php side. Reason is that current timestamp is done on database side, so using the timezone of the database that is completely out of control of the developer, leading to unexpected result.
You can instead add a check in create or update method of crud class to force value to dol_now().

Reply:
@eldy  makes good sense, and it is probably database agnostic as well